ui/pages/confirmations/components/fee-details-component/fee-details-component.js

```diff
@@ -148,13 +136,13 @@ export default function FeeDetailsComponent({
               boldHeadings={false}
             />
           )}
-          {isMultiLayerFeeNetwork && estimatedL1Fees && (
+          {isMultiLayerFeeNetwork && layer1GasFee && (
             <TransactionDetailItem
               detailTitle={t('layer1Fees')}
               detailText={
-                useCurrencyRateCheck && renderTotalDetailText(estimatedL1Fees)
+                useCurrencyRateCheck && renderTotalDetailText(layer1GasFee)
               }
-              detailTotal={renderTotalDetailValue(estimatedL1Fees)}
+              detailTotal={renderTotalDetailValue(layer1GasFee)}
```

Comment:
Just for audit sake, I assume the previous value and our value are both in WEI and hexadecimal?